docs/technical_reference/unit_models/translators/translator_asm1_adm1.rst

```diff
@@ -0,0 +1,194 @@
+ASM1 to ADM1 Translator
+=======================
+
+Introduction
+------------
+
+A link is required to translate between biological and physically- or chemically-mediated processes
+to develop whole-plant modeling of wastewater treatment. This model mediates the interaction between
+the Activated Sludge Model 1 (ASM1)  and the Anaerobic Digestor Model 1 (ADM1).
+
+The model relies on the following key assumptions:
+
+   * supports only liquid phase
+   * supports only ASM1 to ADM1 translations
+
+.. index::
+   pair: watertap.unit_models.translators.translator_asm1_adm1;translator_asm1_adm1
+
+.. currentmodule:: watertap.unit_models.translators.translator_asm1_adm1
+
+Degrees of Freedom
+------------------
+The translator degrees of freedom are the inlet feed state variables:
+
+    * temperature
+    * pressure
+    * volumetric flowrate
+    * solute compositions
+    * cations
+    * anions
+
+Ports
+-----
+
+This model provides two ports:
+
+* inlet
+* outlet
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Time", ":math:`t`", "[0]"
+   "Inlet/outlet", ":math:`x`", "['in', 'out']"
+   "Phases", ":math:`p`", "['Liq']"
+   "Inlet Components", ":math:`j`", "['H2O', 'S_I', 'S_S', 'X_I', 'X_S', 'X_BH', 'X_BA', 'X_P', 'S_O', 'S_NO', 'S_NH', 'S_ND', 'X_ND', 'S_ALK']"
+   "Ion", ":math:`j`", "['S_cat', 'S_an'] \  :sup:`*`"
+   "Outlet Components", ":math:`j`", "['H2O', 'S_su', 'S_aa', 'S_fa', 'S_va', 'S_bu', 'S_pro', 'S_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_I', 'X_c', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'S_cat', 'S_an', 'S_co2']"
+
+**Notes**
+ :sup:`*` Ion" is a subset of "Component" and uses the same symbol j.
+
+.. _Translator_ASM1_ADM1_parameters:
+
+Parameters
+----------
+
+.. csv-table::
+   :header: "Description", "Symbol", "Parameter Name", "Value", "Units"
+
+   "Nitrogen fraction in particulate products", ":math:`i_{xe}`", "i_xe", 0.06, ":math:`\text{dimensionless}`"
+   "Nitrogen fraction in biomass", ":math:`i_{xb}`", "i_xb", 0.08, ":math:`\text{dimensionless}`"
+   "Anaerobic degradable fraction of X_I and X_P", ":math:`f_{xI}`", "f_xI", 0.05, ":math:`\text{dimensionless}`"
```

Comment:
We should either (1) include links to the components tables for ASM1 and ADM1 for people to refer to follow this, or (2) the components tables for both ASM1 and ADM1 could be included directly to more easily follow the acronyms. 

Option (2) would be ideal.

Reply:
In fact, it would be good to do both: reference ASM1 and ADM1 property model documentation while also adding a single components section with tables for ASM1 and ADM1 components and descriptions (or two sections, i.e., ASM1 Components and ADM1 Components)